Templates/TimerTrigger/function.json

```diff
@@ -2,7 +2,7 @@
   "disabled": false,
   "bindings": [
     {
-      "name": "myTimer",
+      "name": "timerinfo",
```

Comment:
The C# templates all seem to use the "my*" naming convention. We should try to be consistent with those.